backend/src/Customer/customer.models.ts

```diff
@@ -0,0 +1,61 @@
+import db from '../shared/dbConnection';
+
+class Customer {
+    public customerId!: number;
```

Comment:
Try not to use the ``!`` operator where it can be avoided.

You can easily get rid of it by adding ``customerId`` to the constructor of the ``Customer`` class, and by modifying the ``addCustomer`` method to accept a name and email instead of a customer object.